charts/osm/crds/multiclusterservice.yaml

```diff
@@ -58,6 +58,9 @@ spec:
                       name:
                         description: Name of the remote cluster
                         type: string
+                      weight:
+                        description: Load balancing weight of the remote cluster in percentage
+                        type: integer
```

Comment:
should we add a check to ensure its a positive integer and if not specified assign a default of 0 ?

Reply:
One correction I added: the weight here is not percentage.

Reply:
Updated minimum to 0, and no limit on maximum, as long as it's in the range of integer